observability/monitoring/configuring-user-workload-monitoring/configuring-performance-and-scalability-uwm.adoc

```diff
@@ -6,7 +6,60 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-TBD
+You can configure the monitoring stack to optimize the performance and scale of your clusters. The following documentation provides information about how to distribute the monitoring components and control the impact of the monitoring stack on CPU and memory resources.
 
+// Using node selectors to move monitoring components
 
+include::modules/monitoring-using-node-selectors-to-move-monitoring-components.adoc[leveloffset=+1]]
 
+[role="_additional-resources"]
+.Additional resources
+
+* TBD
+
+include::modules/monitoring-moving-monitoring-components-to-different-nodes.adoc[leveloffset=+2,tags=**;!CPM;UWM]
+
+[role="_additional-resources"]
+.Additional resources
+
+* TBD
+
+include::modules/monitoring-assigning-tolerations-to-monitoring-components.adoc[leveloffset=+1,tags=**;!CPM;UWM]
+
+[role="_additional-resources"]
+.Additional resources
+
```

Comment:
Hello, I should have clarified this, the links will be filled in in a different PR, because the content from other assemblies still needs to be moved to the new place, which means that if I filled the links now, I would have to keep repairing them later. So I am keeping that for the separate PR. (the user will not see this until all is ready)

Reply:
Ah ok that makes a lot of sense! Thanks for explaining :)